app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class TournamentsController < ApplicationController
+
+  skip_before_action :verify_authenticity_token
+
+  def index
+  end
+
+  def create
+    @company_name = params[:company_name]
+    @start_date = params[:start_date]
+    @end_date = params[:end_date]
+
+    @tournament = Tournament.new(:company_name => params[:company_name], :start_date => params[:start_date], :end_date => params[:end_date])
```

Comment:
When it comes to hashes there is an old syntax (what's written here) and a new syntax, so prefer the new syntax:
```suggestion
    @tournament = Tournament.new(company_name: params[:company_name], start_date: params[:start_date], end_date: params[:end_date])
```
In this syntax the key is automatically a symbol